src/sqlfluff/rules/L049.py

```diff
@@ -0,0 +1,65 @@
+"""Implementation of Rule L006."""
+
+
+from sqlfluff.core.rules.base import BaseRule, LintResult
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
+
+
+@document_fix_compatible
+class Rule_L049(BaseRule):
+    """Comparisons with NULL should use "IS" or "IS NOT".
+
+    | **Anti-pattern**
+    | In this example, the <> is used to check for NULL values'.
+
+    .. code-block:: sql
+
+        SELECT
+            a
+        FROM foo
+        WHERE a <> NULL
```

Comment:
Is `=` the more obvious example?

```suggestion
    | In this example, the '=' is used to check for NULL values'.

    .. code-block:: sql

        SELECT
            a
        FROM foo
        WHERE a = NULL
```

Reply:
Done!